include/openenclave/internal/sgxtypes.h

```diff
@@ -548,8 +548,17 @@ oe_thread_data_t* oe_get_thread_data(void);
 
 #define TD_MAGIC 0xc90afe906c5d19a3
 
+#define OE_THREAD_LOCAL_SPACE (3304)
```

Comment:
Just a question for my  understanding, how did you decide on 3304?

Reply:
This is the free space available in the page that holds the td_t structure.
This limitation of 3304 bytes will be removed by #1161